angular/src/lib/select/select.component.ts

```diff
@@ -212,6 +215,7 @@ export class SelectComponent implements OnInit, AfterContentChecked, AfterConten
       event.removed.forEach(option => option.deselect());
     });
 
+    // tslint:disable-next-line: deprecation
```

Comment:
Is this a good idea? I thought we had the linter for a reason

Reply:
I would tend to agree with Susan on this. Typically we would disable next line in scenarios where there's no other solution.  What's the deprecated method here?